resource-managers/kubernetes/core/pom.xml

```diff
@@ -108,6 +108,14 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-servlet</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jetty</groupId>
+      <artifactId>jetty-http</artifactId>
+    </dependency>
```

Comment:
It's required to only build the kubernetes sub module (much faster compare to a full build), e.g. :

```sh
build/mvn -Pkubernetes -Phadoop-2.6 -pl resource-managers/kubernetes/core install
```

Since jars of jetty artifacts are shaded in the top-level pom.xml, we need to reference them explicitly here, like [what we already did for guava libs](https://github.com/apache-spark-on-k8s/spark/blob/v2.1.0-kubernetes-0.1.0-rc1/resource-managers/kubernetes/core/pom.xml#L88).